pkg/visitor/list.go

```diff
@@ -40,6 +43,12 @@ func ListProjects(ctx context.Context,
 		Filter: filter,
 	})
 	for r, err := it.Next(); err != iterator.Done; r, err = it.Next() {
+		if err != nil && status.Code(err) == codes.Unimplemented && implicitProject != nil {
+			// If the admin service is unavailable, provide a placeholder project.
+			// If the project is invalid, downstream actions will fail.
+			return handler(implicitProject)
```

Comment:
I see: here we don't know the name that we want, so we have to trust the value set in implicitProject.